controls/sarbanes-oxley compliance.json

```diff
@@ -0,0 +1,354 @@
+{
+  "name": "Sarbanes Oxley Compliance",
+  "items": [
+    {
+      "type": "Paragraph",
+      "text": "This checklist draws from Sarbanes-Oxley sections 302 and 404. Use it to perform a periodic review to ensure that compliance measures are in place and in use within the organization. At the conclusion of the audit, the SOX officer should sign off indicating that he/she has verified the information.",
+      "choices": [
+        ""
+      ]
+    },
+    {
+      "type": "SectionBreak",
+      "choices": []
+    },
+    {
+      "type": "Heading",
+      "text": "Establishing safeguards to prevent data tampering. Section 302.2",
+      "choices": [
+        ""
+      ]
+    },
+    {
+      "type": "SectionBreak",
+      "choices": []
+    },
+    {
+      "type": "YesNo",
+      "text": "An ERP system or GRC software is in place, which tracks user logins and access to all computers containing sensitive data.",
+      "choices": [
+        "Yes",
+        " No"
+      ]
+    },
+    {
+      "type": "YesNo",
+      "text": "The systems in place are designed to detect break-in attempts to the organization's computers, databases, fixed and removable storage, and websites.",
+      "choices": [
+        "Yes",
+        " No"
+      ]
+    },
+    {
+      "type": "LongAnswer",
+      "text": "Notes on ERP system or GRC software:",
+      "choices": [
+        ""
+      ]
+    },
+    {
+      "type": "SectionBreak",
+      "choices": []
+    },
+    {
+      "type": "Heading",
+      "text": "Establish safeguards to accurate timeline data. Section 302.3",
+      "choices": [
+        ""
+      ]
+    },
+    {
+      "type": "SectionBreak",
+      "choices": []
+    },
+    {
+      "type": "YesNo",
+      "text": "The ERP system or GRC software timestamps, in real-time, all data received.",
+      "choices": [
+        "Yes",
+        " No"
```

Comment:
Several, but not all, of the the YesNo types have "choices" values. I'm not exactly sure where these are coming from (maybe deprecated?). I don't think they _hurt_ anything but it doesn't look like an exported YesNo item contains them currently and they'll probably just be ignored.